### PR TITLE
Fixed #34512 -- Restored breadcrumbs on admin app index view.

### PR DESCRIPTION
**Pull request by felixxm**
Thanks Adam (ataylor32) for the report.

Regression in 872b61193b013a700ff88cf50f0eb2cf2c266ff7.

ticket-34512

### CODE REVIEW

File: django/contrib/admin/templates/admin/app_index.html [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
+{% block nav-breadcrumbs %}
+  <nav aria-label="{% translate 'Breadcrumbs' %}">
+    <div class="breadcrumbs">
+      <a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+      &rsaquo;
+      {% for app in app_list %}
+        {{ app.name }}
+      {% endfor %}
+    </div>
+  </nav>
```
Comment (shangxiao) [inline django/contrib/admin/templates/admin/app_index.html:16]: Nice 👍 A little more "proper" than what proposed in the ticket. There's some duplication but it's minor.

Sometimes in codebases I like to put in a "if you edit this please also edit this other thing" comment with WET code across files like this... not necessary but just a suggestion 😁

**Review by shangxiao — APPROVED** [review]
😎

**Comment by felixxm** [comment]
buildbot, test on selenium.

File: django/contrib/admin/templates/admin/app_index.html (hunk 1 above)
Comment (felixxm) [reply django/contrib/admin/templates/admin/app_index.html:16]: We have a test that covers this duplication, so it should be fine.

**Comment by felixxm** [comment]
@shangxiao Thanks for checking :+1:

**Pull request merged**